CMakeLists.txt

```diff
@@ -470,8 +470,9 @@ endif()
 add_definitions(-DBOOST_ALL_NO_LIB=1)
 find_package(Boost REQUIRED COMPONENTS ${BOOST_COMPONENTS})
 
-find_path(HALF_INCLUDE_DIR half/half.hpp)
+find_path(HALF_INCLUDE_DIR half.hpp HINTS ${CMAKE_INSTALL_PREFIX}/include ${CMAKE_INSTALL_PREFIX}/include/half )
 message(STATUS "HALF_INCLUDE_DIR: ${HALF_INCLUDE_DIR}")
+string(APPEND CMAKE_CXX_FLAGS " -I${HALF_INCLUDE_DIR}")
```

Comment:
@trixirt @atamazov I am not sure if I like this. I hope that we do not need to modify `CMAKE_CXX_FLAGS` in such a way even though we are only appending to it.